dangerfile.js

```diff
@@ -1,40 +0,0 @@
-import { danger, error, message, schedule, warn } from 'danger';
```

Comment:
`danger` didn't appear to be used anywhere that I could find and the package that we are using is 8 major version out of date with various security issues.